website/docs/r/workspaces_directory.html.markdown

```diff
@@ -3,69 +3,115 @@ subcategory: "WorkSpaces"
 layout: "aws"
 page_title: "AWS: aws_workspaces_directory"
 description: |-
-  Provides a directory registration in AWS WorkSpaces Service.
+  Provides a WorkSpaces directory in AWS WorkSpaces Service.
 ---
 
 # Resource: aws_workspaces_directory
 
-Provides a directory registration in AWS WorkSpaces Service
+Provides a WorkSpaces directory in AWS WorkSpaces Service
 
 ## Example Usage
 
 ```hcl
-resource "aws_vpc" "main" {
+resource aws_vpc example {
   cidr_block = "10.0.0.0/16"
 }
 
-resource "aws_subnet" "private-a" {
-  vpc_id            = aws_vpc.main.id
+resource aws_subnet example_a {
+  vpc_id            = aws_vpc.example.id
   availability_zone = "us-east-1a"
   cidr_block        = "10.0.0.0/24"
 }
 
-resource "aws_subnet" "private-b" {
-  vpc_id            = aws_vpc.main.id
+resource aws_subnet example_b {
+  vpc_id            = aws_vpc.example.id
   availability_zone = "us-east-1b"
   cidr_block        = "10.0.1.0/24"
 }
+resource aws_subnet example_c {
+  vpc_id            = aws_vpc.example.id
+  availability_zone = "us-east-1c"
+  cidr_block        = "10.0.2.0/24"
+}
+
+resource aws_subnet example_d {
+  vpc_id            = aws_vpc.example.id
+  availability_zone = "us-east-1d"
+  cidr_block        = "10.0.3.0/24"
+}
 
-resource "aws_directory_service_directory" "main" {
+
+resource aws_directory_service_directory example {
   name     = "corp.example.com"
   password = "#S1ncerely"
   size     = "Small"
+
   vpc_settings {
-    vpc_id     = aws_vpc.main.id
-    subnet_ids = [aws_subnet.private-a.id, aws_subnet.private-b.id]
+    vpc_id = aws_vpc.example.id
+    subnet_ids = [
+      aws_subnet.example_a.id,
+      aws_subnet.example_b.id
+    ]
   }
 }
 
-resource "aws_workspaces_directory" "main" {
-  directory_id = aws_directory_service_directory.main.id
+resource aws_workspaces_directory example {
```

Comment:
why are all the resource no using "" notation? lets align with all other resources.

Reply:
same for tests

Reply:
If I understand correctly, Terraform Plugin SDK v2 encourages to omit redundant quotes.

- #8950
- #14417

Reply:
I don't think it relevant for resource names/types only on reference.

@ewbankkit / @bflad can you confirm I'm not wrong?

Reply:
Returned quotes back to resource type and name. Thank you for pointing this out.